lib/taskana-core/src/main/java/pro/taskana/classification/internal/models/ClassificationImpl.java

```diff
@@ -16,30 +16,12 @@
 
   public ClassificationImpl() {}
 
-  public ClassificationImpl(ClassificationImpl classification) {
-    this.id = classification.getId();
-    this.key = classification.getKey();
-    this.parentId = classification.getParentId();
-    this.parentKey = classification.getParentKey();
-    this.category = classification.getCategory();
-    this.type = classification.getType();
-    this.domain = classification.getDomain();
-    this.isValidInDomain = classification.getIsValidInDomain();
-    this.created = classification.getCreated();
-    this.modified = classification.getModified();
-    this.name = classification.getName();
-    this.description = classification.getDescription();
-    this.priority = classification.getPriority();
-    this.serviceLevel = classification.getServiceLevel();
-    this.applicationEntryPoint = classification.getApplicationEntryPoint();
-    this.custom1 = classification.getCustom1();
-    this.custom2 = classification.getCustom2();
-    this.custom3 = classification.getCustom3();
-    this.custom4 = classification.getCustom4();
-    this.custom5 = classification.getCustom5();
-    this.custom6 = classification.getCustom6();
-    this.custom7 = classification.getCustom7();
-    this.custom8 = classification.getCustom8();
+  public ClassificationImpl(ClassificationImpl copyFrom) {
```

Comment:
please make this constructor private. The code segment which is using the constructor should use the new `clone` method instead.